src/main/kotlin/it/unibo/alchemist/AlchemistExecutionOptions.kt

```diff
@@ -38,12 +47,20 @@ data class AlchemistExecutionOptions(
     val server: String? = null,
     val parallelism: Int = defaultParallelism,
     val endTime: Double = defaultEndTime,
```

Comment:
We can drop this. `terminators` are available in the configuration file.

Reply:
Removed, see general changes to Loader and Launcher creation. End time is dropped, only terminators are used.